utils/build-mlir-aie.sh

```diff
@@ -37,19 +37,22 @@ mkdir -p $INSTALL_DIR
 cd $BUILD_DIR
 set -o pipefail
 set -e
-cmake -GNinja\
-    -DCMAKE_C_COMPILER=clang \
-    -DCMAKE_CXX_COMPILER=clang++ \
+cmake -GNinja \
+    -DCMAKE_C_COMPILER=clang-12 \
+    -DCMAKE_CXX_COMPILER=clang++-12 \
     -DLLVM_DIR=${LLVM_BUILD_DIR}/lib/cmake/llvm \
     -DMLIR_DIR=${LLVM_BUILD_DIR}/lib/cmake/mlir \
     -DCMAKE_MODULE_PATH=${CMAKEMODULES_DIR}/modulesXilinx \
     -DCMAKE_INSTALL_PREFIX="../${INSTALL_DIR}" \
     -DCMAKE_BUILD_TYPE=Release \
     -DLLVM_ENABLE_ASSERTIONS=ON \
+    -DLLVM_USE_LINKER=lld-12 \
     -DAIE_ENABLE_BINDINGS_PYTHON=ON \
-    "-DAIE_RUNTIME_TARGETS=x86_64;aarch64" \
-    -DAIE_RUNTIME_TEST_TARGET=aarch64 \
-    .. |& tee cmake.log
+    -DAIE_RUNTIME_TARGETS:STRING="x86_64;aarch64" \
+    -DAIE_RUNTIME_TEST_TARGET=x86_64 \
+    -DSysrootAarch64="/group/xrlabs/platforms/vck190-pynq-v2.7/sysroot" \
+    -DVitisSysroot="/group/xrlabs/platforms/vck190-pynq-v2.7/sysroot" \
+    /scratch2/jefff/ipu/pynqMLIR-AIE |& tee cmake.log
```

Comment:
Outsider chiming in! Adding these hard-coded paths, as well as switching clang to clang-12 and lld to lld-12, is causing the build to fail on my machine.

Reply:
Sorry, see https://github.com/Xilinx/mlir-aie/pull/467

Reply:
Oh wonderful, that was quick. Thank you!